packages/nuxt/src/core/runtime/nitro/renderer.ts

```diff
@@ -46,15 +48,22 @@ const getSSRRenderer = lazyCachedFunction(async () => {
   if (!createSSRApp) { throw new Error('Server bundle is not available') }
 
   // Create renderer
-  const renderToString = async (input, context) => {
-    const html = await _renderToString(input, context)
-    return `<div id="__nuxt">${html}</div>`
-  }
-  return createRenderer(createSSRApp, {
+  const renderer = createRenderer(createSSRApp, {
     clientManifest,
     renderToString,
     publicPath: buildAssetsURL()
   })
+
+  async function renderToString (input, context) {
+    const html = await _renderToString(input, context)
+    // In development with vite-node, the manifest is on-demand and will be available after rendering
+    if (process.env.NUXT_VITE_NODE_OPTIONS) {
```

Comment:
This allows tree-shaking this part:

```suggestion
    if (process.dev && process.env.NUXT_VITE_NODE_OPTIONS) {
```